views/viewer.ejs

```diff
@@ -6,15 +6,17 @@
 
+    <script>
+      window.defaultSizes = <%- escapeJson(defaultSizes) %>;
+      window.enableWebSocket = <%- escapeJson(enableWebSocket) %>;
+    </script>
     <%- include('script', { filename: 'viewer.js' }) %>
   </head>
 
   <body>
     <div id="app"></div>
     <script>
       window.chartData = <%- escapeJson(chartData) %>;
-      window.defaultSizes = <%- escapeJson(defaultSizes) %>;
```

Comment:
Can you keep this `window.defaultSizes` as it was, as it seems to be unrelated to your fix?

Reply:
sure, no problem.
> I thought `defaultSizes` is a quite simple variable, so I moved it up.